.vsts-ci.yml

```diff
@@ -133,6 +133,26 @@ stages:
               _PublishType: none
               _SignType: test
 
+    - template: /eng/build.yml
```

Comment:
@mmitche  revert this? Or macos will blocking the build

Reply:
it is running 3 hours if not blocking

Reply:
macOS still works quite a bit of the time, I can add it back in if you want though.